recipes/fredapi/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "fredapi" %}
+{% set version = "0.4.1" %}
```

Comment:
Do you want 0.4.1? If not, let's update this to 0.4.2

Reply:
@isuruf Good catch.  It should be 0.4.2.  Just committed.